src/lib/provable/test/foreign-field-gadgets.unit-test.ts

```diff
@@ -92,7 +92,8 @@ for (let F of fields) {
   let big264 = unreducedForeignField(264, F); // this is the max size supported by our range checks / ffadd
   let big258 = unreducedForeignField(258, F); // rough max size supported by ffmul
 
-  equivalentProvable({ from: [big264, big264], to: big264 })(
+  // addition can fail on two unreduced inputs because we can get x + y - f > 2^264
+  equivalentProvable({ from: [big264, f], to: big264 })(
```

Comment:
we didn't catch this before because of the missing errors on multi range check failures 😬